### PR TITLE
feat: probability filter

### PR DESCRIPTION
**Pull request by joostwmd**
introduced the probability filter feature. 

UI is a slider currently, as it was the most minimal effort element

### CODE REVIEW

File: src/components/organisms/MapBoxMap/MapboxMap.tsx [hunk 1]
```diff
@@ -77,20 +78,33 @@ const MapboxMap: React.FC = () => {
...
             }
         }
 
         return
     }
 
+
+    function handleProbabilityFilterChange(e: React.ChangeEvent<HTMLInputElement>) {
+        for (let timestamp of currentSelectedTimestamps) {
+            const predictionLayerId = `prediction-${timestamp}-${currentAoiId}`
```
Comment (emely3h) [inline src/components/organisms/MapBoxMap/MapboxMap.tsx:101]: this needs to change to:  const predictionLayerId = `prediction-${timestamp}-${currentAoiId}-point`
(I added a heatmap layer but to still have the distinct points I kept the existing point layer, we have now 2 preditcion layers. To distinguish them I added "-heatmap" and  "-point" to the layerId.

**Comment by emely3h** [comment]
the heatmap layer adds an other layer of complexity as the filtering is right now just for the point layer. We either do the same for the heatmap layer or we figure out a way to have the heatmap layer with the distinct points:)

**Comment by emely3h** [comment]
I like the slider, it looks good:) However I would use for the dot the existing gray color that we use for all other buttons and make it a bit smaller.

File: src/components/atoms/ProbabilityFilter/ProbabilityFilter.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+#probability-slider::-ms-fill-upper {
+    background: transparent;
+    border: 2px solid white;
+    border-radius: 4px;
+}
+
+#probability-slider:focus::-ms-fill-upper {
+    background: transparent;
+}
```
Comment (emely3h) [inline src/components/atoms/ProbabilityFilter/ProbabilityFilter.css:101]: do you really need that much css for the styling?
Why are you using so many pseudo elements (ms-fill-upper, moz-range-tracker,...)? I think they all adress different browsers which is not needed I think, we wanna have the same styling for all browsers.

File: src/components/organisms/OEWHeader/OEWHeader.tsx [hunk 3]
```diff
@@ -9,6 +9,8 @@ import MapProjectionButton from '../../atoms/MapProjectionButton/MapProjectionBu
 import { ProbabilityLegend } from '../../atoms/ProbabilityLegend/ProbabilityLegend'
 import DaySelect from '../../molecules/DaySelect/DaySelect'
 import { IRegionData } from '../MapBoxMap/types'
+import { ActionMeta } from 'react-select'
+import { ProbabilityFilter } from '../../atoms/ProbabilityFilter/ProbabilityFilter'
```
Comment (emely3h) [inline src/components/organisms/OEWHeader/OEWHeader.tsx:13]: same import twice, can be removed

File: src/services/mapService.tsx [hunk 4]
```diff
@@ -71,11 +71,10 @@ export async function fetchPredictions(
         }
 
         const predictions: FeatureCollection<Point, IPredProperties> = await response.json()
-
+        console.log('fetchPredictions', predictions)
```
Comment (emely3h) [inline src/services/mapService.tsx:74]: can be removed

File: src/services/predictionLayerService.tsx [hunk 5]
```diff
@@ -28,6 +28,7 @@ export function addPredictionLayer(
     aoiId: number,
     predictionQueryData: FeatureCollection<Point, IPredProperties>,
 ) {
+    console.log('adding prediction layer', predictionQueryData)
```
Comment (emely3h) [inline src/services/predictionLayerService.tsx:31]: can be removed

**Comment by tomADC443** [comment]
does the slider now also works if I select more then one timestamp? :)

**Comment by tomADC443** [comment]
I will take over this feature :)

**Pull request closed without merging**